tools/chapel-py/src/core-types.cpp

```diff
@@ -196,6 +207,28 @@ PyObject* ContextObject_get_pyi_file(ContextObject *self, PyObject* args) {
     ss << std::endl;
   #include "method-tables.h"
 
+  #define ENSURE_ALL_CLASSES(NODE) \
+    if(generated.find(tagToUserFacingStringTable[asttags::NODE]) == generated.end()) { \
+      ss << "class " << tagToUserFacingStringTable[asttags::NODE] << "("; \
+      generated.insert(tagToUserFacingStringTable[asttags::NODE]); \
```

Comment:
Doesn't seem like we need this line since it won't visit a type more than once (why would ENSURE_ALL_CLASSES be called again?)